src/frontend-v2/README.md

```diff
@@ -0,0 +1,27 @@
+# Frontend
```

Comment:
This is boilerplate, right? Try to follow the pattern of the other service READMEs

Reply:
Copied and modified the old frontend README which we can update going forward as functionality in the new frontend is added.